logstash-core/api/lib/app/root.rb

```diff
@@ -0,0 +1,15 @@
+# encoding: utf-8
+require "app"
+
+module LogStash::Api
+  class Root < BaseApp
+
+    get "/" do
+      content = { "name" => "Logstash API",
+                  "version" => { "number" => "0.1.0" },
+                  "tag" => "You know, for logs" }
```

Comment:
well, it's all perspective.

When I say log, I mean (time, data). Most people think of logs as being server logs, so maybe it's fair to try a different term. I dunno.